WordPress/src/main/java/org/wordpress/android/ui/themes/ThemeBrowserAdapter.java

```diff
@@ -65,6 +68,7 @@ public View newView(Context context, Cursor cursor, ViewGroup parent) {
 
         ThemeViewHolder themeViewHolder = new ThemeViewHolder(view);
         view.setTag(themeViewHolder);
+        mViewWidth = parent.getWidth();
```

Comment:
Good point, I'll change it so the width is passed in from the Fragment where I can divide the width by the number of columns